01-Authorization-MVC/gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
```

8.4 is the latest version and would be better for Java 21 compatibility. https://docs.gradle.org/8.4/release-notes.html

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-bin.zip
```